data/scripting/field_animations.lua

```diff
@@ -132,7 +143,7 @@ function reveal_concentric(plr, center, max_radius, hide, delay)
 end
 
 -- RST
--- .. function:: hide_concentric(player, center, max_radius, delay)
+-- .. function:: hide_concentric(player, center, max_radius, delay, permanent)
```

Comment:
here too mark the argument as optional
```suggestion
-- .. function:: hide_concentric(player, center, max_radius, delay[, permanent=false])
```